websocket-server/metrics.ts

```diff
@@ -31,40 +32,40 @@ const recentDisconnects = new Map< string, number >(); // connection_id -> disco
  * ------------------------------------------------------------
  */
 const connectedClientsGauge = new Gauge( {
-	name: 'websocket_connected_clients',
+	name: `${ METRICS_NAMESPACE }_connected_clients`,
```

Comment:
not necessarily related to this PR, but wondering if it would be helpful to have a histogram of connected clients, with buckets `[ 1, 5, 10, 20, 50, 100, 200, 500, 1000 ]`?

Reply:
I think you can do `max_over_time(gauge_metric[time_period])` for that